simulationTool/components/SimulationTool.vue

```diff
@@ -92,15 +98,37 @@ export default {
             <ProcessList
                 v-if="mode === 'process-list'"
                 @selected="selectProcess"
+                @close="() => setMode('home-panel')"

```

Comment:
Are these `close` events emitted form the components already?

Reply:
I removed them for now